test/data/BUILD

```diff
@@ -30,5 +36,6 @@ filegroup(
 checkstyle_test(
     name = "checkstyle",
     include = glob(["*"]),
+    exclude = glob(["sample_file_structure/*", "sample_file_structure/*/*"]),
```

Comment:
Our files within sample_file_structure and its subdirectories are just empty files. Putting licence headers in them doesn't really make sense.